mmv1/third_party/terraform/data_sources/data_source_google_firebase_web_app.go.erb

```diff
@@ -12,7 +12,7 @@ func dataSourceGoogleFirebaseWebApp() *schema.Resource {
 	dsSchema := datasourceSchemaFromResourceSchema(resourceFirebaseWebApp().Schema)
 
 	// Set 'Required' schema elements
-	addRequiredFieldsToSchema(dsSchema, "app_id")
```

Comment:
Could you remove the whitespace changes? It doesn't look like this file has any changes of interest.

Reply:
@mraouffouad just following up because I saw you made updates, but the whitespace changes have not been addressed yet. For example, this file should have no changes in the final diff.